core/server/web/api/canary/admin/routes.js

```diff
@@ -212,6 +177,9 @@ module.exports = function apiRoutes() {
     );
     router.del('/session', mw.authAdminApi, http(apiCanary.session.delete));
 
+    // ## Identity
+    router.get('/identity', mw.authAdminApi, http(apiCanary.identity.read));
```

Comment:
@allouis what are your thoughts on naming this endpoint?

Reply:
I think it's cool

Reply:
My only concern is that it deviates from the general convention where resource endpoints are named as plurals. Also not sure if we should just return a string in a response body or also follow the convention here and return something like:
```
{
  identities: [{
    token
  }]
}
```
Wdyt?

Reply:
Yeah that's true, probs best to stick with convention :+1: